### DISCUSSION TITLE
Added

### DISCUSSION THREAD

**Discussion by PrimalXY** [20GotoTen/es-theme-retrofix#20] [Ideas]
Hello, could you add higher quality fanarts?
I would also appreciate adding other systems like the Amiga 1200 or general Arcade.
Good job 😊

**Comment by PrimalXY** [comment]
Hi, I see that you’re updating the repository. Among other things, you’ve included the Amiga 1200 😊. I have all the FBNeo and MAME games in the automatic Arcade game collection. Many themes include it—could you also include an icon and a background for it?

**Reply by 20GotoTen** [reply]
Hi! Yes, I'll add that in this week. If there are any more requests, I'm happy to consider them. Thank you for using the theme!

**Comment by PrimalXY** [comment]
Thank you very much for the Arcade folder! I was wondering if it might be possible to hide the Batocera logo over Arcade. Also, if it’s not too much trouble 😅, it would be great to have the option to display game details, such as playtime.
